app/src/main/java/ch/epfl/sweng/swenggolf/profile/Badge.java

```diff
@@ -103,8 +103,8 @@ public static String getColor(int points) {
      * @param points the amount of points of the user
      * @return the level of the corresponding Badge
      */
-    private static int computeLevel(int points) {
-        int level = (points + 1) / 7;
-        return (level > MAX_LEVEL) ? MAX_LEVEL : level;
+    public static int computeLevel(int points) {
+        int level = (points+1)/7;
```

Comment:
I know it's not you who'd written that, but wouldn't it be better to avoid these magic numbers?

Reply:
Yes, you're right! 
In fact, I wonder where this computation comes from and if it is the best for the current way to collect points... If someone likes your answer it makes you gain 10 points which corresponds to more than one level; I'll ask @dingepfl what he had in mind about it.

Reply:
Oh sorry I didn't see this and I set it to 20, I guess it will just be a bit harder to level up ;)

Reply:
I think 20 is even better, so please leave it like that! ;)